### PR TITLE
feat: add v1 contents of installation section

### PR DESCRIPTION
**Pull request by ghost**
Adding the contents for version 1.0 in the "Installation" section to provide installation procedures.

This contents was diverted from [the version 1.0 document](https://gitlab.com/autowarefoundation/autoware_reference_design) and I revised them to adjust for the current document structure. After merging this PR, the contents of "How to run Autoware" and "How to run simulators" sections in Start guide chapter will be added.

Signed-off-by: Daisuke Hashimoto <daisuke.hashimoto@tier4.jp>

### CODE REVIEW

**Comment by NaoyoriTanzawa** [comment]
@d-hashimoto Could you point out the link to check the organized pages with markdown?
I do not find it this time...

**Comment by ghost** [comment]
@NaoyoriTanzawa Sorry for the inconvenience. Please refer to the following link:
https://autowarefoundation.github.io/open-ad-kit-docs/pr-13/

**Review by NaoyoriTanzawa — APPROVED** [review]

**Comment by NaoyoriTanzawa** [comment]
@d-hashimoto I understand this is just for installation section and it is reconfigured from the original one. I just approved it.

**Comment by ghost** [comment]
@NaoyoriTanzawa Thank you. After passing "spell-check-differential" with https://github.com/tier4/autoware-spell-check-dict/pull/303#, I'll merge it.

File: docs/start-guide/installation/getting-started-pcu.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+   For Linux users, you could use "cutecom" to connect to PCU.
+
+   ```console
+   sudo apt install cutecom
+   cutecom
+   ```
+
+   Please set parameters as follows, `Device` shall be chosen based on your host PC.
+
+   ![PCU Interface Front](images/getting-started-pcu/ttyusb-parameters.png)
```
Comment (kenji-miyake) [inline docs/start-guide/installation/getting-started-pcu.md]:
```suggestion
   ![PCU Interface Front](images/getting-started-pcu/tty-usb-parameters.png)
```

**Pull request merged**